elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -30,73 +14,66 @@ div.pl-order-blocks-bottom {
 }
 
 
-ul.dropzone {
-  /* This style applies to answer dropzone */
-  background-color: #FFA; /* Yellow */
-  min-height: 30px;
+ul.pl-order-blocks-connected-sortable li:hover{
+  cursor: move; /* For UX: The answer tiles shows a 'grab' cursor on mouse over */
+  outline: 2px solid blue;
+  outline-offset: -2px;
 }
 
-
-ul.pl-order-blocks-connected-sortable li, ul.dropzone li{
-  /* This style applies to the answer draggable tiles themselves */
-  background-color: #EFEFEF; /* Grey */
-  border: 2px solid #A9A9A9;
-  border-radius: 4px;
-  margin: 0 10px 10px 10px;
-  padding: 5px;
-  font-size: 1.2em;
-
-  display: block;
-
-  /* Show overflow text in the answer draggable tiles */
-  white-space: normal;
-  height: auto;
-  max-height: none;
-  overflow: visible;
+li.ui-state-highlight { 
+  /*This styling applies to the placeholder answer tile that appears when the user is dragging */
+  height: 3em !important; 
+  border: none !important;
+  list-style-type: none;
+  background-color: #90a4ae !important;
+  min-width: 3em !important;
 }
```

Comment:
Do all these `!important`s need to be here? there are a lot of them in this PR and I'm not sure all (or any) of them are necessary.

Reply:
Yes. This is because the element uses bootstrap CSS as base, and we uses `!important` to override and adjust the bootstrap design CSS so the element wouldn't appear excessively "flat" and are more visible.